examples/simple-connect/package.json

```diff
@@ -7,12 +7,18 @@
     "babel-polyfill": "^6.26.0",
     "ethers": "^4.0.43",
     "jsbi": "^3.1.1",
-    "react": "^16.12.0",
-    "react-dom": "^16.12.0",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "token-amount": "^0.1.0",
     "use-wallet": "link:../.."
   },
+  "alias": {
+    "react": "../../node_modules/react",
+    "react-dom": "../../node_modules/react-dom"
+  },
   "devDependencies": {
-    "@babel/core": "^7.8.3",
+    "@babel/core": "^7.0.0-0",
```

Comment:
👀 I thought this version was for moving from babel 6 to babel 7?

Reply:
Not sure what happened here, I moved it to `^7.11.1` :+1: